src/coreclr/tools/Common/Compiler/DependencyAnalysis/CompilerComparer.cs

```diff
@@ -11,6 +11,8 @@ namespace ILCompiler.DependencyAnalysis
 {
     public class CompilerComparer : TypeSystemComparer, IComparer<ISortableNode>
     {
+        public static new CompilerComparer Instance { get; } = new CompilerComparer();
```

Comment:
We have number of places that do `new CompilerComparer`. Switch them to call this instead?